api/v3/Order.php

```diff
@@ -74,8 +74,8 @@ function civicrm_api3_order_create($params) {
   civicrm_api3_verify_one_mandatory($params, NULL, ['line_items', 'total_amount']);
   $entity = NULL;
   $entityIds = [];
-  $contributionStatus = $params['contribution_status_id'] ?? NULL;
-  if ($contributionStatus !== 'Pending' && 'Pending' !== CRM_Core_PseudoConstant::getName('CRM_Contribute_BAO_Contribution', 'contribution_status_id', $contributionStatus)) {
+  $params['contribution_status_id'] = $params['contribution_status_id'] ?? 'Pending';
+  if ($params['contribution_status_id'] !== 'Pending' && 'Pending' !== CRM_Core_PseudoConstant::getName('CRM_Contribute_BAO_Contribution', 'contribution_status_id', $params['contribution_status_id'])) {
     CRM_Core_Error::deprecatedFunctionWarning("Creating a Order with a status other than pending is deprecated. Currently empty defaults to 'Completed' so as a transition not passing in 'Pending' is deprecated. You can chain payment creation e.g civicrm_api3('Order', 'create', ['blah' => 'blah', 'contribution_status_id' => 'Pending', 'api.Payment.create => ['total_amount' => 5]]");
```

Comment:
This warning needs updating too

Reply:
what to - just don't pass in contribution_status_id? Or should we just remove it & always hard-code to Pending now?

Reply:
Technically this is now incorrect `Currently empty defaults to 'Completed'` as it now defaults to `Pending`.
However I would be quite happy to hardcode to Pending and do away with allowing any other status. That simplifies things from a maintenance perspective and forces anyone using it "wrongly" to update their code. @artfulrobot has probably used Order.create more than most and may have some insight?

Reply:
@mattwire OK - it should be accurate now - & we can decide whether we want to go further & deprecate passing it in at all at some point